k8s/gcp/digger/postgresql.yaml

```
@@ -7,9 +7,15 @@ spec:
   teamId: "digger"
   volume:
     size: 1Gi
-    selector:
-      matchLabels:
-        cloud.google.com/gke-spot: "true"
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: cloud.google.com/gke-spot
+            operator: In
+            values:
+            - "true"
   numberOfInstances: 2
```

追加要件すみません。
こちら修正お願いします。

```suggestion
  numberOfInstances: 1
```